Source/WebInspectorUI/UserInterface/Workers/Formatter/ESTreeWalker.js

```diff
@@ -288,11 +288,13 @@ ESTreeWalker = class ESTreeWalker
         case "DebuggerStatement":
         case "EmptyStatement":
         case "Identifier":
-        case "Import":
```

Comment:
You'll want to update `ScriptSyntaxTree.js` with the new node types and ensure that the AST is walkable. If we're removing `Import` we probably need to add `ImportExpression`, `ChainExpression`, and `PrivateIdentifier`.

* `WI.ScriptSyntaxTree.NodeType ` enum should have new types <https://github.com/WebKit/WebKit/blob/main/Source/WebInspectorUI/UserInterface/Models/ScriptSyntaxTree.js#L1054>
* `WI.ScriptSyntaxTree.prototype._recurse` to handle the new types <https://github.com/WebKit/WebKit/blob/main/Source/WebInspectorUI/UserInterface/Models/ScriptSyntaxTree.js#L310>
* `WI.ScriptSyntaxTree._createInternalSyntaxTree` to handle the new types.

Reply:
Thank you for bringing these up. This is now handled and tested in the latest PR.